Packs/MicrosoftTeams/ReleaseNotes/1_4_70.md

```diff
@@ -0,0 +1,7 @@
+
+
+#### Integrations
+
+##### Microsoft Teams via Webhook
+
+- Updated the integration to adapt to the new format of Microsoft Teams webhook by using workflows instead of the incoming webhook. for more information- [link](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-via-webhook).
```

Comment:
```suggestion
Updated the integration to adapt to the new format of Microsoft Teams webhook by using workflows instead of the incoming webhook. For more information, see [link](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-via-webhook).
```